### ISSUE TITLE
Display custom Icon in OSX High Sierra 10.13.3

### ISSUE THREAD

**Issue by RationalCoding** [tomas/dialog#12]
The default AppleScript icon is shown instead of anything informative. Is there anyway to show something relevant to my application?

```javascript
var dialog = require('dialog')
dialog.info('Hello there')
```

![image](https://user-images.githubusercontent.com/14932492/35763037-f9c7791c-0870-11e8-994f-6bafa7823ec3.png)

**Comment by tomas** [comment]
Yes, we could support passing a custom icon path (or using one of the built-ins, e.g `with icon 0` or `with icon caution`). Happy to merge a patch if you're up to it!
